scripts/prepare_default_model.py

```
@@ -183,25 +232,38 @@ def validate_model():
             b1 = data['b1']
             w2 = data['w2']
             b2 = data['b2']
+            w3 = data['w3']
+            b3 = data['b3']
```

Ruff (0.14.10)

Local variable `b3` is assigned to but never used

Remove assignment to unused variable `b3`

(F841)